tests/testapps/MinApp/MinApp.vsimporter/MinApp-WinStore10/project.json

```diff
@@ -0,0 +1,17 @@
+{
+  "dependencies": {
+    "WinObjC.Frameworks": "*",
```

Comment:
You'll want to move to PackageReference instead. Additionally, you'll want to add all packages (except tools and packaging) here to handle cases where you update a single pacakge that isn't a leaf node and want to test local updates.